__tests__/pathToDotNotation.test.js

```diff
@@ -0,0 +1,13 @@
+const pathToDotNotation = require('../build.js').pathToDotNotation
+
+test('valid token', () => {
```

Comment:
Could we wrap this in a describe block? It will help to provide that additional context if we add verbose logging and coverage reports...

```suggestion
describe('Token formatters: PascalCase', () => {
  it('renders a valid token name in PascalCase', () => {
```